scss/forms/_form-control.scss

```diff
@@ -23,6 +23,14 @@
   @include box-shadow($input-box-shadow);
   @include transition($input-transition);
 
+  &[type="file"] {
```

Comment:
while not related to this, mind if we slip in `&[type="color"]` here as well?

Reply:
otherwise, to keep it clean, happy to wait for this to be merged and then adding it in a new PR that's targeted